packages/core/src/StatusIcon/StatusIcon.styles.tsx

```diff
@@ -0,0 +1,58 @@
+import { createClasses } from "@hitachivantara/uikit-react-utils";
+import { theme } from "@hitachivantara/uikit-styles";
+
+const notificationMap = {
+  success: "positive",
+  warning: "warning",
+  error: "negative",
+  default: "info",
+  info: "info",
+  accent: "accent",
+} as const;
+
+export const { staticClasses, useClasses } = createClasses("HvStatusIcon", {
+  root: {
+    borderRadius: theme.radii.round,
+    lineHeight: 0,
+    flexShrink: 0,
+    padding: theme.space.xxs,
+    "--icsize": "auto",
+    " .HvIconContainer-root": {
+      padding: 0,
+      " .HvIconContainer-xs": { fontSize: 16, padding: 4 },
+      " .HvIconContainer-sm": { fontSize: 24, padding: 4 },
```

Comment:
Where are these values coming from? The spec for `HvStatusIcon` seems to be consistent with `HvIconContainer`, which is `20px` for `sm`
![image](https://github.com/user-attachments/assets/9cf9933c-d029-42f2-9744-ccc942b82ace)